src/core/lib/gprpp/fork.h

```diff
@@ -58,7 +59,7 @@ class Fork {
   // reset the polling engine's internal state.
   static void SetResetChildPollingEngineFunc(
       child_postfork_func reset_child_polling_engine);
-  static child_postfork_func GetResetChildPollingEngineFunc();
+  static std::vector<child_postfork_func> GetResetChildPollingEngineFunc();
```

Comment:
I think we can return this by const ref instead of by copy, `static const std::vector<child_postfork_func>& GetResetChildPollingEngineFunc()`

Reply:
Ah yeah. I meant to go back and do that. Fixed.